lib/metasploit/framework/spec/threads/suite.rb

```diff
@@ -12,7 +12,7 @@ module Suite
           #
 
           # Number of allowed threads when threads are counted in `after(:suite)` or `before(:suite)`
-          EXPECTED_THREAD_COUNT_AROUND_SUITE = ENV['REMOTE_DB'] ? 3 : 2
+          EXPECTED_THREAD_COUNT_AROUND_SUITE = ENV['REMOTE_DB'] ? 4 : 3
```

Comment:
Just for visibility, what's the extra thread that's being added?

Reply:
I believe it's the new Rails thread manager that handles `ActionCable`, `Rails.application.executor`, the new thread-safe code loader and so on. I can check more thoroughly later if you want.